### PR TITLE
Include Omero-channels-metadata update in import-ome-zarr task

### PR DESCRIPTION
**Pull request by tcompa**
## Checklist before merging
- [x] I added an appropriate entry to `CHANGELOG.md`

### CODE REVIEW

**Comment by tcompa** [comment]
Last missing point from #578:
- [ ]  Assess whether we also need to add values for Window and/or colormap, if missing;

**Comment by tcompa** [comment]
This is ready for a review, and it will be the last contribution to the upcoming v0.13.0.

**Comment by tcompa** [comment]
(see here for a recap: https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/578#issuecomment-1770273392)

File: fractal_tasks_core/lib_channels.py [hunk 1]
```diff
@@ -339,3 +341,134 @@ def define_omero_channels(
...
+    Returns:
+        New list of Fractal-compatible Omero-channel dictionaries
+    """
+    new_channels = deepcopy(old_channels)
+    existing_wavelength_ids: list[str] = []
+    handled_channels = []
+
+    default_colors = ["00FFFF", "FF00FF", "FFFF00"]
+
+    def _get_next_color() -> str:
```
Comment (jluethi) [inline fractal_tasks_core/lib_channels.py:395]: Oh, this is elegant! I wanted that, but couldn't think of a good way of implementing it, so didn't suggest it =D

File: fractal_tasks_core/lib_channels.py [hunk 2]
```diff
@@ -339,3 +341,134 @@ def define_omero_channels(
...
+        try:
+            return default_colors.pop(0)
+        except IndexError:
+            return "808080"
+
+    # Channels that with "wavelength_id"
```
Comment (jluethi) [inline fractal_tasks_core/lib_channels.py:401]: Wording suggestion: Channels that contain the key "wavelength_id"

File: fractal_tasks_core/tasks/import_ome_zarr.py [hunk 3]
```diff
@@ -100,6 +106,46 @@ def _process_single_image(
...
 
+    # Update Omero-channels metadata
+    if update_omero_metadata:
+        # Extract number of channels from zarr array
+        try:
+            channel_axis_index = image_meta.axes_names.index("c")
+        except ValueError:
+            raise NotImplementedError(
+                f"Existing axes ({image_meta.axes_names}) do not "
+                'include "c".'
```
Comment (jluethi) [inline fractal_tasks_core/tasks/import_ome_zarr.py:117]: Let's link to the issue 150 here: https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/150
OME-Zarrs without channel axis aren't currently supported in Fractal. We are working on improving our axis flexibility, follow along in https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/150

**Review by jluethi — APPROVED** [review]
Looks great! Besides the 2 minor remarks on a comment & a message in the NotImplementedError, looks good to merge from my side!

**Comment by tcompa** [comment]
Thanks for the review, merging now

**Pull request merged**